firmware_mod/config/telegram.conf.dist

```diff
@@ -3,5 +3,5 @@
 ############################################################
 
 # Add your Telegram credentials here (these are just examples)
-apiToken="23132160:AAFrSEwzJ-X-64tLnD8JJ$JoAxWIMg$$w"
-userChatId="30985132"
+apiToken="yuorapitoken"
+userChatId="yourchatid"
```

Comment:
I think those fantasy credentials convey more information about what is expected to be put there. Please leave as it was.